ods_ci/tests/Tests/500__jupyterhub/custom-image.robot

```diff
@@ -84,38 +85,30 @@ Test Bad Image URL
     ${IMG_URL}=  Set Variable  quay.io/RandomName/RandomImage:v1.2.3
     Set Global Variable  ${IMG_URL}  ${IMG_URL}
     Create Custom Image
-    RHODS Notification Drawer Should Contain  Unable to add notebook image ${IMG_NAME}
-    ${IMG_URL}=  Set Variable  ${OG_URL}
-    Set Global Variable  ${IMG_URL}  ${IMG_URL}
-    Reset Image Name
-
-Test Bad Image Import
-    [Documentation]  Import a broken image and confirm it is disabled
-    ...    in the JH spawner page
-    [Tags]    Sanity    Tier1
-    ...       ODS-1364
-    ${OG_URL}=  Set Variable  ${IMG_URL}
-    ${IMG_URL}=  Set Variable  randomstring
-    Set Global Variable  ${IMG_URL}  ${IMG_URL}
-    Create Custom Image
-    RHODS Notification Drawer Should Contain
-    ...  Unable to add notebook image ${IMG_NAME}
+    Wait Until Page Contains    Invalid repository URL: ${IMG_URL}
+    # Since the image cannot be created, we need to cancel the modal window now
+    Click Button    ${GENERIC_CANCEL_BTN_XP}
+    [Teardown]  Bad Image URL Teardown  orig_url=${OG_URL}
```

Comment:
why did you merge the two tests?

Reply:
It's described in the commit message:

```
Since the behaviour of the custom image creation now is that you cannot
create an image with a wrong image location at all, this test simply
duplicates what is covered in `Test Bad Image URL` already.
```

Truth is that later I was able to create an image with image location like this:
```
image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/s2i-generic-data-science-notebook
```
but such image reported error `"image" not found` and couldn't be enabled and used in JupyterHub anyway.

Thus I think that this scenario doesn't make sense anymore.